server/chat-commands/room-settings.ts

```diff
@@ -15,6 +15,26 @@ const SLOWCHAT_MINIMUM = 2;
 const SLOWCHAT_MAXIMUM = 60;
 const SLOWCHAT_USER_REQUIREMENT = 10;
 
+export const sections = [
+	'officialrooms', 'officialtiers', 'communityprojects', 'languages', 'gaming', 'leisure', 'lifehobbies', 'onsitegames',
+] as const;
+
+export type RoomSection = typeof sections[number];
+
+export const RoomSections: {sectionNames: {[k in RoomSection]: string}, sections: readonly RoomSection[]} = {
+	sectionNames: {
+		officialrooms: 'Official rooms',
+		officialtiers: 'Official tiers',
+		communityprojects: 'Community projects',
```

Comment:
I think these names make Official Rooms and Official Tiers sound more related than they actually are, and Upper Staff agreed.

New names:

```suggestion
		official: 'Official',
		smogontiers: 'Smogon tiers',
		communityprojects: 'Community projects',
```

Reply:
I also wanted "Smogon formats" and "Community projects and formats" but I couldn't build consensus for it.